Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -976,35 +1000,79 @@ void ReflectometryReductionOneAuto2::applyPolarizationCorrection(
   }
 }
 
-/**
- * Sum transmission workspaces that belong to a workspace group
- * @param transGroup : The transmission group containing the transmission runs
- * @return :: A workspace pointer containing the sum of transmission
- * workspaces
- */
-MatrixWorkspace_sptr ReflectometryReductionOneAuto2::sumTransmissionWorkspaces(
-    WorkspaceGroup_sptr &transGroup) {
-
-  const std::string transSum = "trans_sum";
-  Workspace_sptr sumWS = transGroup->getItem(0)->clone();
-
-  /// For this step to appear in the history of the output workspaces I need
-  /// to set child to false and work with the ADS
-  auto plusAlg = createChildAlgorithm("Plus");
-  plusAlg->setChild(false);
-  plusAlg->initialize();
-
-  for (size_t item = 1; item < transGroup->size(); item++) {
-    plusAlg->setProperty("LHSWorkspace", sumWS);
-    plusAlg->setProperty("RHSWorkspace", transGroup->getItem(item));
-    plusAlg->setProperty("OutputWorkspace", transSum);
-    plusAlg->execute();
-    sumWS = AnalysisDataService::Instance().retrieve(transSum);
+MatrixWorkspace_sptr ReflectometryReductionOneAuto2::getFloodWorkspace() {
```

Comment:
Doxygen missing here?

Reply:
I think the method's name clearly indicates what the method is doing. A comment like `/// Get flood workspace` won't make it clearer.

Reply:
ok